src/scc_hypervisor_collector/cli/__init__.py

```diff
@@ -38,8 +82,14 @@ def main(argv: Optional[Sequence[str]] = None) -> None:
                         help="The config directory to check for YAML "
                              "config files.")
     parser.add_argument('-C', '--check', action='store_true',
-                        help="Check the configuration data only, "
-                             "reporting any errors.")
+                        help="Check the configuration data "
+                             "only reporting any errors.")
+    default_log_destination = f"{os.path.expanduser('~')}/" \
+                              f"scc-hypervisor-collector.log"
+    parser.add_argument('-L', '--logfile', action='store',
+                        default=default_log_destination,
+                        help=f"path to logfile. "
```

Comment:
The initial part of the help message doesn't need to be an f-string as there is no embedded `{...}` to be processed.

```suggestion
                        help="path to logfile. "
```